### ISSUE TITLE
Train/Val ResNET loss not decreasing?

### ISSUE THREAD

**Issue by Declan-Curran1** [trybushenko/Fairface-Face-Classification#1]
Is the '14012024-model-train-resnet.ipynb' file the best for model training here? It appears the base version of this file is not converging when training?

Training step:
Epoch = 1, Loss = 5.802036762237549
Age accuracy = 0.1124000288205202	Race accuracy = 0.14477988327689315	Gender accuracy = 0.5013041285395201
Age loss = 2.1974997520446777	Race loss = 1.9461579322814941	Gender Loss = 0.6933397650718689
Validation step:
Epoch = 1, Loss = 5.8145994672308055
Age accuracy = 0.010605798605106923	Race accuracy = 0.14260187907083982	Gender accuracy = 0.5251023113724134
Age loss = 2.2061822414398193	Race loss = 1.9458523988723755	Gender Loss = 0.6927870512008667
End of Epoch: 1 	Training Loss: 5.802036762237549 	Validation Loss: 5.8145994672308055
Validation loss decreased (inf --> 5.8145994672308055).  Saving model ...
Training step:
Epoch = 2, Loss = 5.802229881286621
Age accuracy = 0.11107428489084228	Race accuracy = 0.14342531882700482	Gender accuracy = 0.5002377692917357
Age loss = 2.1976091861724854	Race loss = 1.9461939334869385	Gender Loss = 0.6933053731918335
Validation step:
Epoch = 2, Loss = 5.8206918750510095
Age accuracy = 0.010605798605106923	Race accuracy = 0.10692258919822469	Gender accuracy = 0.4748976886275866
Age loss = 2.208580732345581	Race loss = 1.9475785493850708	Gender Loss = 0.6942843794822693
End of Epoch: 2 	Training Loss: 5.802229881286621 	Validation Loss: 5.8206918750510095
Training step:
Epoch = 3, Loss = 5.801960468292236
Age accuracy = 0.11166510555515527	Race accuracy = 0.14100439512933208	Gender accuracy = 0.500180128251315
Age loss = 2.1975228786468506	Race loss = 1.9461039304733276	Gender Loss = 0.6932769417762756
Validation step:
Epoch = 3, Loss = 5.804586905388736
Age accuracy = 0.12000691682517724	Race accuracy = 0.1261167790650758	Gender accuracy = 0.5251023113724134
Age loss = 2.200453281402588	Race loss = 1.9446431398391724	Gender Loss = 0.6930475234985352
End of Epoch: 3 	Training Loss: 5.801960468292236 	Validation Loss: 5.804586905388736
Validation loss decreased (5.8145994672308055 --> 5.804586905388736).  Saving model ...
Training step:
Epoch = 4, Loss = 5.801973819732666
Age accuracy = 0.110757259168528	Race accuracy = 0.14382880610995027	Gender accuracy = 0.49984869226889544
Age loss = 2.1974925994873047	Race loss = 1.94614577293396	Gender Loss = 0.6933017373085022
Validation step:
Epoch = 4, Loss = 5.808516580982923
Age accuracy = 0.12000691682517724	Race accuracy = 0.142313678021788	Gender accuracy = 0.5251023113724134
Age loss = 2.203519582748413	Race loss = 1.9440497159957886	Gender Loss = 0.6931748986244202
End of Epoch: 4 	Training Loss: 5.801973819732666 	Validation Loss: 5.808516580982923
Training step:
Epoch = 5, Loss = 5.801921367645264
Age accuracy = 0.11170833633547085	Race accuracy = 0.14214280567764248	Gender accuracy = 0.49984869226889544
Age loss = 2.197399854660034	Race loss = 1.9461880922317505	Gender Loss = 0.693291187286377
Validation step:
Epoch = 5, Loss = 5.81061860796354
Age accuracy = 0.031471554556458586	Race accuracy = 0.19015505216438988	Gender accuracy = 0.4748976886275866
Age loss = 2.2043395042419434	Race loss = 1.944483995437622	Gender Loss = 0.6941079497337341
End of Epoch: 5 	Training Loss: 5.801921367645264 	Validation Loss: 5.81061860796354
Training step:
Epoch = 6, Loss = 5.801885604858398
Age accuracy = 0.11322141364651632	Race accuracy = 0.14392967793068664	Gender accuracy = 0.500626846314576
Age loss = 2.1974453926086426	Race loss = 1.94611656665802	Gender Loss = 0.6932830214500427
Validation step:
Epoch = 6, Loss = 5.796576607783864
Age accuracy = 0.010605798605106923	Race accuracy = 0.10692258919822469	Gender accuracy = 0.4748976886275866
Age loss = 2.193572521209717	Race loss = 1.9462932348251343	Gender Loss = 0.6942772269248962
End of Epoch: 6 	Training Loss: 5.801885604858398 	Validation Loss: 5.796576607783864
Validation loss decreased (5.804586905388736 --> 5.796576607783864).  Saving model ...
Training step:
Epoch = 7, Loss = 5.801883220672607
Age accuracy = 0.1104546437063189	Race accuracy = 0.1429353699834282	Gender accuracy = 0.49980546148857985
Age loss = 2.1974904537200928	Race loss = 1.9460514783859253	Gender Loss = 0.6932955384254456
Validation step:
Epoch = 7, Loss = 5.784874168692817
Age accuracy = 0.07297250561992046	Race accuracy = 0.15049858781485964	Gender accuracy = 0.4748976886275866
Age loss = 2.1856496334075928	Race loss = 1.9465848207473755	Gender Loss = 0.6934460401535034
End of Epoch: 7 	Training Loss: 5.801883220672607 	Validation Loss: 5.784874168692817
Validation loss decreased (5.796576607783864 --> 5.784874168692817).  Saving model ...
Training step:
Epoch = 8, Loss = 5.8019232749938965
Age accuracy = 0.11167951581526046	Race accuracy = 0.1434829598674256	Gender accuracy = 0.5014914619208877
Age loss = 2.197479009628296	Race loss = 1.946100115776062	Gender Loss = 0.6932862997055054
Validation step:
Epoch = 8, Loss = 5.7948975603175095
Age accuracy = 0.22081964378350338	Race accuracy = 0.14139143466482218	Gender accuracy = 0.4748976886275866
Age loss = 2.190944194793701	Race loss = 1.948289155960083	Gender Loss = 0.6940061450004578
End of Epoch: 8 	Training Loss: 5.8019232749938965 	Validation Loss: 5.7948975603175095
Training step:
Epoch = 9, Loss = 5.801998615264893
Age accuracy = 0.11081490020894877	Race accuracy = 0.14456372937531523	Gender accuracy = 0.49734130701059154
Age loss = 2.197505235671997	Race loss = 1.9461036920547485	Gender Loss = 0.6933379173278809
Validation step:
Epoch = 9, Loss = 5.800650586229923
Age accuracy = 0.01942475070609257	Race accuracy = 0.10692258919822469	Gender accuracy = 0.5251023113724134
Age loss = 2.1953132152557373	Race loss = 1.9478343725204468	Gender Loss = 0.6930271983146667
End of Epoch: 9 	Training Loss: 5.801998615264893 	Validation Loss: 5.800650586229923
Training step:
Epoch = 10, Loss = 5.801913261413574
Age accuracy = 0.11114633619136825	Race accuracy = 0.14476547301678797	Gender accuracy = 0.49904171770300454
Age loss = 2.19747257232666	Race loss = 1.9461315870285034	Gender Loss = 0.6932669878005981
Validation step:
Epoch = 10, Loss = 5.807739642195468
Age accuracy = 0.12317712836474724	Race accuracy = 0.1261167790650758	Gender accuracy = 0.4748976886275866
Age loss = 2.201741933822632	Race loss = 1.945624589920044	Gender Loss = 0.6932500600814819
End of Epoch: 10 	Training Loss: 5.801913261413574 	Validation Loss: 5.807739642195468
Training step:
Epoch = 11, Loss = 5.801982879638672
Age accuracy = 0.10937387419842928	Race accuracy = 0.14424670365300093	Gender accuracy = 0.5001945385114202
Age loss = 2.1975486278533936	Race loss = 1.946088194847107	Gender Loss = 0.6932854056358337
Validation step:
Epoch = 11, Loss = 5.798065119861525
Age accuracy = 0.11983399619574615	Race accuracy = 0.19015505216438988	Gender accuracy = 0.5251023113724134
Age loss = 2.196916103363037	Race loss = 1.9436601400375366	Gender Loss = 0.6924655437469482
End of Epoch: 11 	Training Loss: 5.801982879638672 	Validation Loss: 5.798065119861525
Training step:
Epoch = 12, Loss = 5.802195072174072
Age accuracy = 0.11235679804020463	Race accuracy = 0.14122054903091	Gender accuracy = 0.4963614093234383
Age loss = 2.197591781616211	Race loss = 1.9461485147476196	Gender Loss = 0.6933515071868896
Validation step:
Epoch = 12, Loss = 5.791464866286396
Age accuracy = 0.2724076315637789	Race accuracy = 0.1861778776874748	Gender accuracy = 0.4748976886275866
Age loss = 2.1912262439727783	Race loss = 1.9451972246170044	Gender Loss = 0.6932528018951416
End of Epoch: 12 	Training Loss: 5.802195072174072 	Validation Loss: 5.791464866286396
Training step:
Epoch = 13, Loss = 5.801934719085693
Age accuracy = 0.11077166942863319	Race accuracy = 0.14355501116795158	Gender accuracy = 0.49989192304921104
Age loss = 2.197481870651245	Race loss = 1.9460887908935547	Gender Loss = 0.6932955384254456
Validation step:
Epoch = 13, Loss = 5.794742890778467
Age accuracy = 0.031471554556458586	Race accuracy = 0.142313678021788	Gender accuracy = 0.4748976886275866
Age loss = 2.190687656402588	Race loss = 1.948514461517334	Gender Loss = 0.6937652230262756
End of Epoch: 13 	Training Loss: 5.801934719085693 	Validation Loss: 5.794742890778467
Training step:
Epoch = 14, Loss = 5.801802158355713
Age accuracy = 0.11166510555515527	Race accuracy = 0.14639383240867498	Gender accuracy = 0.49813387131637726
Age loss = 2.1974806785583496	Race loss = 1.9460160732269287	Gender Loss = 0.6932754516601562
Validation step:
Epoch = 14, Loss = 5.812563684501978
Age accuracy = 0.12000691682517724	Race accuracy = 0.142313678021788	Gender accuracy = 0.4748976886275866
Age loss = 2.2045466899871826	Race loss = 1.946078896522522	Gender Loss = 0.693846583366394
End of Epoch: 14 	Training Loss: 5.801802158355713 	Validation Loss: 5.812563684501978
Training step:
Epoch = 15, Loss = 5.801937580108643
Age accuracy = 0.11085813098926435	Race accuracy = 0.14273362634195547	Gender accuracy = 0.5019525902442539
Age loss = 2.197463035583496	Race loss = 1.946146845817566	Gender Loss = 0.6932321786880493
Validation step:
Epoch = 15, Loss = 5.80069272566254
Age accuracy = 0.22081964378350338	Race accuracy = 0.14260187907083982	Gender accuracy = 0.4748976886275866
Age loss = 2.1955347061157227	Race loss = 1.9473178386688232	Gender Loss = 0.6940711140632629
End of Epoch: 15 	Training Loss: 5.801937580108643 	Validation Loss: 5.80069272566254
Training step:
Epoch = 16, Loss = 5.801820278167725
Age accuracy = 0.1115930542546293	Race accuracy = 0.14189783125585417	Gender accuracy = 0.5038403343180344
Age loss = 2.1973929405212402	Race loss = 1.946173071861267	Gender Loss = 0.6931878924369812
Validation step:
Epoch = 16, Loss = 5.810340507188517
Age accuracy = 0.01942475070609257	Race accuracy = 0.14260187907083982	Gender accuracy = 0.5251023113724134
Age loss = 2.2023258209228516	Race loss = 1.9473588466644287	Gender Loss = 0.692268967628479
End of Epoch: 16 	Training Loss: 5.801820278167725 	Validation Loss: 5.810340507188517
